sources/world/README.md

```diff
@@ -1,45 +1,144 @@
-# Administrative Divisions
+# World datasets
+
+**Note** commands assume a copy of the [Natural Earth Vector repository](https://github.com/nvkelso/natural-earth-vector) is located on this folder as `naturalearth`.
```

Comment:
This is not quite accurate as some of the Natural Earth States and Provinces are incorrect. I wonder if we should publish our own fork of Natural Earth with our corrections?

Reply:
yeah, I've been thinking about that as well, or at least to place a copy of the regions dataset somewhere in this repo to make it self contained.